README.md

```diff
@@ -18,6 +18,8 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
   - [Vue MDC Adapter](https://github.com/stasson/vue-mdc-adapter): MDC Web Integration for Vue.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters).)
   - [Material Components Vue](https://github.com/matsp/material-components-vue): MDC Web Integration for Vue.js (using [vanilla components](./docs/integrating-into-frameworks.md#the-simple-approach-wrapping-mdc-web-vanilla-components))
+  - [Material Components React](https://github.com/material-components/material-components-web-react): MDC Web Integration for React.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters))
+  - [Material Web Components](https://github.com/material-components/material-components-web-components): MDC Web Integration for Web Components (using [vanilla components](./docs/integrating-into-frameworks.md#the-simple-approach-wrapping-mdc-web-vanilla-components))
```

Comment:
What's the value in the self-referential link?

Reply:
are you referring to the Material Web Components link? That is the polymer repo

Reply:
MY BAD!!!!! I completely missed that. The name is so similar.